game/js/items.js

```diff
@@ -1,81 +1,83 @@
 const gold = [
-    {
-        id: 1,
-        name: "gold coin",
-        type: "gold",
-        attack: 0,
-        defense: 0,
-        value: 1
-    }
-]
+  {
+    id: 1,
+    name: "gold coin",
+    type: "gold",
+    attack: 0,
+    defense: 0,
+    value: 1,
+  },
+];
 
 const weapons = [
-    {
-        id: 1000,
-        name: "Small Knife",
-        type: "weapon",
-        attack: 1,
-        value: 5
-    },
-    {
-        id: 1001,
-        name: "Short Sword",
-        type: "weapon",
-        attack: 2,
-        value: 10
-    },
-    {
-        id: 1002,
-        name: "Diego's Club",
-        type: "weapon",
-        attack: 4,
-        value: 20
-    },
-    {
-        id: 1003,
-        name: "Katana",
-        type: "weapon",
-        attack: 5,
-        value: 45
-    },
-    {
-        id: 1004,
-        name: "Scorpion's Claw",
-        type: "weapon",
-        attack: 8,
-        value: 120
-    },
-    {
-        id: 1005,
-        name: "Staff of Falerga",
-        type: "weapon",
-        attack: 12,
-        value: 180
-    }
-]
+  {
+    id: 1000,
+    name: "Small Knife",
+    type: "weapon",
+    attack: 1,
+    value: 5,
+  },
+  {
+    id: 1001,
+    name: "Short Sword",
+    type: "weapon",
+    attack: 2,
+    value: 10,
+  },
+  {
+    id: 1002,
+    name: "Diego's Club",
+    type: "weapon",
+    attack: 4,
+    value: 20,
+  },
+  {
+    id: 1003,
+    name: "Katana",
+    type: "weapon",
+    attack: 5,
+    value: 45,
+  },
+  {
+    id: 1004,
+    name: "Scorpion's Claw",
+    type: "weapon",
+    attack: 8,
+    value: 120,
+  },
+  {
+    id: 1005,
+    name: "Staff of Falerga",
+    type: "weapon",
+    attack: 12,
+    value: 180,
+  },
+];
 
 const shields = [
-    {
-        id: 2000,
-        name: "Wooden Shield",
-        type: "shield",
-        defense: 2,
-        value: 10
-    },
-    {
-        id: 2001,
-        name: "Iron Shield",
-        type: "shield",
-        defense: 4,
-        value: 18
-    },
-    {
-        id: 2002,
-        name: "Shield of Dispute",
-        type: "shield",
-        defense: 7,
-        value: 30
-    },
-]
+  {
+    id: 2000,
+    name: "Wooden Shield",
+    type: "shield",
+    defense: 2,
+    value: 10,
+  },
+  {
+    id: 2001,
+    name: "Iron Shield",
+    type: "shield",
+    defense: 4,
+    value: 18,
+  },
+  {
+    id: 2002,
+    name: "Shield of Dispute",
+    type: "shield",
+    defense: 7,
+    value: 30,
+  },
+];
 
-const ALL_ITEMS = gold.concat(weapons, shields)
+const merchantItems = [...shields, ...weapons];
```

Comment:
Not sure if I want all the items that drop from monsters to be available at the shop. I think we could make items that only drop from monsters and items that can only be bought at the shop. So maybe we should create a new array `merchantItems` from the zero and not by using the current items. What do you think?

Reply:
Tottaly agree, i think it would be a better experience if we have unique items for merchant. I was using this array just as a mock to test the shop and forgot to create new items for shop hahahaha. Ty for the review.